src/praisonai/praisonai/n8n/preview.py

```
@@ -0,0 +1,207 @@
+    
+    # Validate file exists
+    yaml_file = Path(yaml_path)
+    if not yaml_file.exists():
+        raise FileNotFoundError(f"Workflow file not found: {yaml_path}")
+    
+    # Load and parse YAML
+    try:
+        import yaml as yaml_lib
+        with open(yaml_file, 'r') as f:
+            yaml_content = yaml_lib.safe_load(f)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)